src/groovy/org/pih/warehouse/api/Putaway.groovy

```diff
@@ -34,6 +35,14 @@ class Putaway {
         orderedBy(nullable:true)
     }
 
+    List<PutawayItem> getPutawayItems() {
+        if (sortBy == "currentBins") {
+           return putawayItems.sort { a,b -> a?.currentBins <=> b?.currentBins }
```

Comment:
I think you can change it to a?."{sortBy}" <=> b?."{sortBy}" then you just need to check if sortBy is not empty